### ISSUE TITLE
[feature] Consider alignment of field sizes against font size

### ISSUE THREAD

**Issue by archenroot** [pgmodeler/pgmodeler#875]
Hi, 

thanks again for releasing "unlocked" pgModeller for currently unsupported version of Postgres 9.6 I use, for my needs it works without any issue so far (otherwise I will report :-) ).

This issue is more or less related to better support of scaled fonts on high resolution desktops (4K and more). For example when I enter the Column name on Table form, the displayed column name cannot be seen completely.

I believe this can be handled on global level of window drawing.

This is not blocking issue.

Ladislav

**Comment by kb3ien** [comment]
Agreed, frequently a lot of horizontal space is allocated to the last column, but there is not much too see, showing more of the first column would be nice.
